gulpfile.js

```diff
@@ -519,11 +519,11 @@ gulp.task("build:test", gulp.series("clean", "copy", "test", "tfx:test"));
  * =========================
  */
 gulp.task("deploy:vsix:sonarqube", () => {
-  if (process.env.CIRRUS_BRANCH !== "master" && process.env.CIRRUS_PR === "false") {
+  if (process.env.CIRRUS_BRANCH !== "master" && ! process.env.CIRRUS_PR) {
     gutil.log("Not on master nor PR, skip deploy:vsix");
     return gutil.noop;
   }
-  if (process.env.CIRRUS_PR === "true" && process.env.DEPLOY_PULL_REQUEST === "false") {
+  if (process.env.CIRRUS_PR && process.env.DEPLOY_PULL_REQUEST === "") {
     gutil.log("On PR, but artifacts should not be deployed, skip deploy:vsix");
     return gutil.noop;
   }
```

Comment:
This code was incorrect, but likely we never encountered this case hence why it was silently existing